### PR TITLE
*: tiny clean up

### PR DESCRIPTION
**Pull request by BusyJay**
- exclude more directories to build smaller package
- run clippy via cargo clippy instead, so we don't have "*" version in Cargo.toml, which is prohibited by crates.io

### CODE REVIEW

File: grpc-sys/Cargo.toml [hunk 1]
```diff
@@ -11,8 +11,21 @@ description = "FFI bindings to gRPC c core library"
 categories = ["external-ffi-bindings", "network-programming"]
 build = "build.rs"
 exclude = [
+    "grpc/doc/*",
```
Comment (overvenus) [inline grpc-sys/Cargo.toml]: Do it require grpc/ ?
 - .github/
 - .vscode/
 - bazel/
 - summerofcode/
 - src/benchmark/
 - src/csharp/
 - etc...

Comment (BusyJay) [inline grpc-sys/Cargo.toml]: Yes. But these directories only occupy small spaces but requiring a long list to exclude. And the project can't compile without including `src/csharp`, although we don't use any `src/csharp` code here.

Comment (siddontang) [inline grpc-sys/Cargo.toml]: > And the project can't compile without including src/csharp

Why do we depend on csharp?

Comment (BusyJay) [inline grpc-sys/Cargo.toml]: CMake require src/csharp to finish configuration.

**Comment by overvenus** [comment]
LGTM

**Comment by siddontang** [comment]
CI failed @BusyJay

**Pull request merged**